src/mongocxx/test/write_concern.cpp

```diff
@@ -35,8 +35,7 @@ TEST_CASE("a default write_concern", "[write_concern]") {
     }
 
     SECTION("will require confirmation from just the primary or standalone mongod") {
-        // TODO: fix me
-        REQUIRE(-2 == wc.nodes());
+        REQUIRE(static_cast<std::int32_t>(write_concern::w::kDefault) == wc.nodes());
```

Comment:
if you want to be fancy here... you can use std::underlying_type instead of explicitly saying std::int32_t